woodwork/table_accessor.py

```diff
@@ -57,37 +57,41 @@ def init(self, index=None, time_index=None, logical_types=None, make_index=False
             use_standard_tags (bool, optional): If True, will add standard semantic tags to columns based
                 specified logical type for the column. Defaults to True.
             column_descriptions (dict[str -> str], optional): Dictionary mapping column names to column descriptions.
+            schema (Woodwork.Schema, optional): Typing information to use for the DataFrame instead of performing inference.
+                Any other arguments provided will be ignored.
         '''
-        _validate_accessor_params(self._dataframe, index, make_index, time_index, logical_types)
-
-        if make_index:
-            _make_index(self._dataframe, index)
+        _validate_accessor_params(self._dataframe, index, make_index, time_index, logical_types, schema)
+        if schema is not None:
+            self._schema = schema
+        else:
```

Comment:
Right now any extra arguments provided beyond `schema` will be ignored when a schema is passed in. I'm not sure this is ideal, but I think we're going to promote using `df.ww.sample` instead of calling `df.sample` and then needing to directly call the init, so it's probably not worth restricting which parameters you can pass in.

Reply:
Should/could we add a validation check in `_validate_accessor_params` to confirm that if `schema` is not None, that all the other params are none and either raise a Warning or Error instead of silently ignoring? Probably not high priority at the moment, but maybe something we could do eventually.

Reply:
We definitely could. I think it'd make a lot of sense (and probably more sense than quietly ignoring the non-schema params--maybe we just raise a warning?), but figured I'd keep things simple for now

Reply:
Seems fine to keep simple for now and tackle this later - maybe capture in an issue so we don't forget?

Reply:
created #523